WordPress/src/main/res/layout/reader_post_detail_header_view_new.xml

```diff
@@ -23,13 +23,12 @@
 
     <org.wordpress.android.ui.reader.views.ReaderFollowButton
         android:id="@+id/header_follow_button"
-        style="@style/Reader.Follow.Button.PostDetail"
+        style="@style/Reader.Follow.Button.New"
```

Comment:
I believe that after this change, the `@style/Reader.Follow.Button.PostDetail` is technically not being used anywhere. It still has a couple of usages (screenshot below) but the old style (`Reader.Follow.Button`) doesn't really match what exists in prod (icon is smaller) so I wonder if it's worth having it different or just adjust the old header layout to use the new button style anyway. (same as https://github.com/wordpress-mobile/WordPress-Android/pull/19248#discussion_r1336394941)

![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/18f5df8a-f791-4403-8b04-00269e9b8261)